CHANGELOG

```diff
@@ -1,3 +1,23 @@
+* Tue Aug 24 2021 Trevor Vaughan <tvaughan@onyxpoint.com> - 4.16.0
+- Added
+  - simp::puppetdb::disable_update_checking to disable default analytics in
+    accordance with NIST guidance
+  - PuppetDB now sets UseCodeCacheFlushing by default
+- Changed
+  - Migrated from camptocamp/kmod to puppet/kmod
+- Fixed
+  - Corrected the HeapDumpOnOutOfMemoryError setting for PuppetDB
+  - Ensure that nsswitch SSSD options for sudoers do not stop on files
+  - Do not include the auditors sudo user specification if the aliases have not
+    been included
+  - Add the following to sudoers defaults
+    - !visiblepw
+    - always_set_home
+    - match_group_by_gid
+    - always_query_group_plugin
+  - Ensured that users setting `simp_options::ldap` to `false` would not be
```

Comment:
Looks like the change that introduced simp::sssd::client::ldap_server_type was also 4.16.0, so I don't think this change needs to be called out.

Reply:
@lnemsick-simp Is it worth chonking up the runners again to take it out?